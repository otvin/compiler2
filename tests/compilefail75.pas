program compilefail75(output);
{try to writeln() an array that isn't a string type}


var r:array[1..7] of boolean;


begin
	r[1] := true;
	writeln(r);
end.
