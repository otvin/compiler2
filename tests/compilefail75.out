[1mtests/compilefail75.pas:10:5: [0m[1m[91merror:[0m 'r' is of type that cannot be displayed by writeln()
    10 | 	writeln(r);